CMakeLists.txt

```diff
@@ -523,7 +523,7 @@ if(AVIF_CODEC_RAV1E)
 
     # Unfortunately, rav1e requires a few more libraries
     if(WIN32)
-        set(AVIF_PLATFORM_LIBRARIES ${AVIF_PLATFORM_LIBRARIES} legacy_stdio_definitions.lib ntdll.lib advapi32.lib userenv.lib ws2_32.lib bcrypt.lib msvcrt.lib)
+        set(AVIF_PLATFORM_LIBRARIES ${AVIF_PLATFORM_LIBRARIES} ntdll.lib advapi32.lib userenv.lib ws2_32.lib bcrypt.lib msvcrt.lib)
```

Comment:
Vincent: I think `msvcrt.lib` should also be removed.

If you have the time, please remove both `advapi32.lib` and `msvcrt.lib`, and only add them back if .github/workflows/ci-windows.yml fails. Thanks.

Reply:
Sure, done